userCode/main.py

```diff
@@ -585,13 +586,32 @@ def nquads_to_renci(
 
 
 @schedule(
-    cron_schedule="@daily",
+    cron_schedule="@weekly",
     job=harvest_job,
-    default_status=DefaultScheduleStatus.STOPPED,
+    default_status=DefaultScheduleStatus.STOPPED
+    if RUNNING_AS_TEST_OR_DEV
```

Comment:
This is a fn since some of the necesarry check env vars arent read until runtime
```suggestion
    if RUNNING_AS_TEST_OR_DEV()
```